src/__tests__/lib/containers/queryUtils.test.ts

```diff
@@ -3,8 +3,8 @@ import { SynapseConstants } from '../../../lib/utils'
 import syn16787123Json from '../../../mocks/query/syn16787123.json'
 import { QueryResultBundle } from 'src/lib/utils/synapseTypes/Table/QueryResultBundle.js'
 import { cloneDeep } from 'lodash-es'
-import { isFacetAvailable } from '../../../lib/utils/functions/queryUtils'
-import { FacetColumnResult } from '../../../lib/utils/synapseTypes'
+import { isFacetAvailable, isSingleNotSetValue } from '../../../lib/utils/functions/queryUtils'
+import { FacetColumnResult, FacetColumnResultValues } from '../../../lib/utils/synapseTypes'
```

Comment:
Looks like there was an issue with merging upstream changes, as these should not appear to be new